application/src/main/java/org/togetherjava/tjbot/features/mediaonly/MediaOnlyChannelListener.java

```diff
@@ -48,22 +48,18 @@ public void onMessageReceived(MessageReceivedEvent event) {
         }
 
         if (messageHasNoMediaAttached(message)) {
-            message.suppressEmbeds(true).flatMap(any -> dmUser(message)).queue((res)->{
-                        message.delete().queue();
-                    },
-                    new ErrorHandler()
-                            .handle(ErrorResponse.CANNOT_SEND_TO_USER,
-                                    (err)->{
-                                        if (event.getAuthor().isBot()) {
-                                            return;
-                                        }
-                                        message.delete().flatMap(any -> warnUser(message)).queue((res)->{
-                                            res.delete().queueAfter(1, TimeUnit.MINUTES);
-                                        },(error)->{
-
-                                        });
-                                    })
-            );
+            message.suppressEmbeds(true).flatMap(any -> dmUser(message)).queue((res) -> {
```

Comment:
what the supressEmbeds is for?

Reply:
I wqated AuditableRestAction<Void> object in return so I used it

Reply:
not like that

Reply:
then any other simple approach?

Reply:
i mean u do want to get AuditableRestAction but not by supressing embeds but by deleting the message, dont user random method for no reason

Reply:
ok I did resolve it in Submission 6 and it runs all the tests successfully